src/cascade/input_data/db/demographics.py

```diff
@@ -30,20 +31,38 @@ def get_locations(execution_context):
     ]
 
 
-def age_groups_to_ranges(execution_context, data):
+def age_groups_to_ranges(execution_context, data, keep_age_group_id=False):
     groups = get_age_groups(execution_context)
-    with_group = data.merge(groups, on="age_group_id").drop(columns="age_group_id")
+    with_group = data.merge(groups, on="age_group_id")
+    if not keep_age_group_id:
+        with_group = with_group.drop(columns="age_group_id")
     return with_group.rename(columns={"age_group_years_start": "age_lower", "age_group_years_end": "age_upper"})
 
 
-def get_all_age_spans(execution_context):
+def get_mean_years(execution_context):
+    """
+    Get the mean year estimate for age groups. This is the mean age beyond the
+    start of the age group for people in a given age group.
+
```

Comment:
Maybe add a numerical example here.  I am not familiar with the get_life_table, though it sounds like a nice one!

Are you saying in the comments from the other PR, that if an age group is 5-10, this might return 2.5?

get_years_from_lower_age_to_mean_age

Reply:
Ok, that makes sense to me.